File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;
import java.lang.*;

class Solution {
    public String solution(String s) {
        String[] str = s.split(" ");
        
        int max = Integer.MIN_VALUE;
        int min = Integer.MAX_VALUE;
        
        
        for(int i = 0; i < str.length; i++) {
            int num = Integer.parseInt(str[i]);
            
            min = Math.min(num, min);
            max = Math.max(num, max);
            
        }
        
        return min + " " + max;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.js
function solution(s) {
    let min_value = Number.MAX_SAFE_INTEGER;
    let max_value = Number.MIN_SAFE_INTEGER;
    
    s.split(" ").forEach(num => {
        num = Number(num);
        
        if(num > max_value) {
            max_value = num;    
        }
        
        if(num < min_value) {
            min_value = num;
        }
   })

    return min_value + " " + max_value;
}